module/zfs/metaslab.c

```diff
@@ -2169,7 +2169,7 @@ metaslab_potentially_evict(metaslab_class_t *mc)
 	uint64_t allmem = arc_all_memory();
 	uint64_t inuse = spl_kmem_cache_inuse(zfs_btree_leaf_cache);
 	uint64_t size =	spl_kmem_cache_entry_size(zfs_btree_leaf_cache);
-	int tries = 0;
+	uint32_t tries = 0;
```

Comment:
Why uint32_t here, not uint_t if multilist_get_num_sublists() returns unsigned int?

Reply:
It is an artifact of my treatment of `uint32_t` as a short hand for `unsigned int`. I will change it.